libs/ui-toolkit/modals/src/lib/ui-modal/simple-modal/simple-modal.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="icon-container">
+  <img class="icon-close" src="/assets/img/close.svg" (click)="closeDialog()"/>
+</div>
+<header mat-dialog-title>
+  <h2 [innerHtml]="data?.titleText"></h2>
+</header>
+<section mat-dialog-content>
+  <img *ngIf="!!data.bodyImage" class="mat-dialog-content-image" [src]="data?.bodyImage">
+  <p *ngIf="!!data?.bodyText" class="mat-dialog-content-question" [innerHtml]="data?.bodyText"></p>
+  <ng-container *ngIf="data.socialMedia">
```

Comment:
`socialMedia` is an array, right? Empty arrays are truthy objects, so instead of testing its existence, you should test if its length is greater than zero